src/main/java/seedu/address/model/item/field/Level.java

```diff
@@ -0,0 +1,20 @@
+package seedu.address.model.item.field;
+
+/**
+ *  Represents a Skill proficiency level
```

Comment:
```suggestion
 *  Represents a Skill proficiency level.
```
Just to be consistent with the other files (at least the one I have been working on)
Was just reading the Javadoc